lithops/util/ssh_client.py

```diff
@@ -29,13 +30,19 @@ def create_client(self, timeout=2):
         """
         Crate the SSH client connection
         """
-        self.ssh_client = paramiko.SSHClient()
-        self.ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        self.ssh_client.connect(self.ip_address, **self.ssh_credentials,
-                                timeout=timeout, banner_timeout=200,
-                                allow_agent=False)
+        try:
+            self.ssh_client = paramiko.SSHClient()
+            self.ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+            self.ssh_client.connect(self.ip_address, **self.ssh_credentials,
+                                    timeout=timeout, banner_timeout=200,
+                                    allow_agent=False)
 
-        logger.debug("{} ssh client created".format(self.ip_address))
+            logger.debug("{} ssh client created".format(self.ip_address))
+        except Exception as e:
+            pk = self.ssh_credentials.get('key_filename')
+            if pk and str(e) == 'Authentication failed.':
```

Comment:
If wrong private key specified i see exception:
`"Bad authentication type; allowed types: ['publickey']"`

Reply:
Do you updated your code with https://github.com/lithops-cloud/lithops/pull/863/files ?